keps/sig-node/753-sidecar-containers/README.md

```diff
@@ -1297,18 +1604,28 @@ These goals will help you determine what you need to measure (SLIs) in the next
 question.
 -->
 
+- sidecar init containers are running and restarted as expected
```

Comment:
SLOs are effectively metrics+thresholds

Can you please update this section and formulate it more in the context of SLIs below?